superset/security/manager.py

```diff
@@ -244,7 +244,7 @@ def can_access_database(self, database: Union["Database", "DruidCluster"]) -> bo
         return (
             self.can_access_all_datasources()
             or self.can_access_all_databases()
-            or self.can_access("database_access", database.perm)
+            or self.can_access("database_access", database.perm)  # type: ignore
```

Comment:
I'm surprised type checking has to be muted. Apparently mypy isn't comfortable with hybrid properties?

Reply:
I think it struggles to identify which method this refers to.